net-mgmt/pfSense-pkg-lldpd/Makefile

```diff
@@ -0,0 +1,53 @@
+# $FreeBSD$
+
+PORTNAME=	pfSense-pkg-lldpd
+PORTVERSION=	0.9.4
+CATEGORIES=	net-mgmt
+MASTER_SITES=	# empty
+DISTFILES=	# empty
+EXTRACT_ONLY=	# empty
+
+MAINTAINER=	coreteam@pfsense.org
+COMMENT=	802.1ab Link Layer Discovery Protocol (LLDP) daemon
+
+LICENSE=	APACHE20
+
+RUN_DEPENDS=	${LOCALBASE}/sbin/lldpd:net-mgmt/lldpd \
+		${LOCALBASE}/sbin/lldpcli:net-mgmt/lldpd
+
+NO_BUILD=	yes
+NO_MTREE=	yes
+
+SUB_FILES=	pkg-install pkg-deinstall
+SUB_LIST=	PORTNAME=${PORTNAME}
+
+do-extract:
+	${MKDIR} ${WRKSRC}
+
+do-install:
+	${MKDIR} ${STAGEDIR}${PREFIX}/pkg
+	${MKDIR} ${STAGEDIR}${PREFIX}/pkg/lldpd
```

Comment:
${MKDIR} is expanded to `mkdir -p` so when this line is executed /pkg is also created. Line 28 can be removed